client/src/containers/CategoryForm.js

```diff
@@ -1,34 +1,38 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
+import { Redirect } from 'react-router-dom';
 import * as actions from "../actions";
 
 class CategoryForm extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
+    this.state = {
+      redirect: false
+    };
 
     this.selectCategory = this.selectCategory.bind(this);
   }
 
   selectCategory(event) {
     event.preventDefault();
     const categorySelected = event.target.category.value;
-
     this.props.changeCategory(categorySelected);
+    this.props.changeSymptoms(categorySelected);
+    this.setState({ redirect: true });
```

Comment:
Hmmm interesting so you guys are still using redirect here while Link is prefable ( cause you want to navigate to a different page ) but you are also dispatching actions here so im not really sure what to say :thinking:

Reply:
@Karyum ok cool! So refactored the landing page and the Symptoms page to use the Link tag! Much easier you are right, I'm only leaving the anchor tags in the Categories page because we are not using a traditional button and anyway the Link tag was causing some issues with the images and stuff. I refactored the Landing component back to a functional one now that we dont need the redirect state so it's much less code and cleaner. Hope you will be happy with the changes 😄